src/SFA.DAS.Forecasting.Application/Shared/Services/EmployerDatabaseService.cs

```diff
@@ -101,5 +103,37 @@ public class EmployerDatabaseService : BaseRepository, IEmployerDatabaseService
                 throw;
             }
         }
+
+        public async Task<List<long>> GetEmployersWithPayments(int year, int month)
+        {
+            const string sql = "SELECT distinct" +
+                               "[AccountId]" +
+                               "from [employer_financial].[Payment] p " +
+                               "left join [employer_financial].[Accounttransfers] acct on p.AccountId = acct.ReceiverAccountId and p.ApprenticeshipId = acct.ApprenticeshipId and p.PeriodEnd = acct.PeriodEnd " +
```

Comment:
For the minimal performance gain, should re-arrange the order of that to 
```
acct.ReceiverAccountId = p.AccountId
```
but wont lose any sleep if you dont 💃 (same for below)